static/css/components/rating-form.less

```diff
@@ -14,16 +14,48 @@
   cursor: pointer;
   margin: 0 auto;
 }
-
+// allows smooth scrolling after clicking on anchor link
+html {
+  scroll-behavior: smooth;
+}
 .review {
   display: flex;
+  scroll-margin-top: 100px;
   .review-btn {
     border-left: 1px solid @lighter-grey;
     width: 35px;
     height: 35px;
   }
+  // initial setting before clicking on anchor link
+  border: 1px solid transparent;
+  border-radius: 5px;
+  -webkit-transition: all 1s linear;
+  position: sticky;
+  top: 0;
+}
+// highlighting settings
+@keyframes highlight {
+  0% {
+    border: 1px solid transparent;
+    box-shadow: 0 0 10px transparent;
+  }
+  50% {
+    border: 1px solid @link-blue;
+    box-shadow: 0 0 10px @link-blue;
+  }
+  100% {
+    border: 1px solid transparent;
+    box-shadow: 0 0 10px transparent;
+  }
+}
+// smooth scrolling and highlighting area after clicking on anchor link
+.review:target {
+  animation: highlight 3s;
+}
+//changes mouse cursor to 'pointer'
+.avg-ratings{
+  cursor: pointer;
```

Comment:
```suggestion
```
Removing this, as we don't want people to think that this element is interactive.  We just want to direct people who mistakenly thought that they could add a star rating by clicking here to the correct place.